en/drafts/originals/creating-guis-in-python-for-digital-humanities-projects.md

```diff
@@ -53,40 +40,27 @@ PDF file. The main goal is to learn the basics of creating GUIs. By the end of t
+{% include figure.html filename="CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS1.png" alt="Visual description of figure image" caption="Figure 1. Final Design of the User Interface" %}
 
 ## Prerequisites and Tools
 
 The following instructions work on macOS, Windows, and Linux machines.
 
-You should have some familiarity with the command line interface on your system. For Linux and Mac users, you can learn
-this through Programming Historian's [Introduction to the Bash Command Line](https://programminghistorian.org/en/lessons/intro-to-bash).
-Windows users can learn about the Power Shell with Programming Historian's [Introduction to the Windows Command Line with PowerShell](https://programminghistorian.org/en/lessons/intro-to-powershell).
+You should have some familiarity with the command line interface on your system. For Linux and Mac users, you can learn this through _Programming Historian_'s [Introduction to the Bash Command Line](/en/lessons/intro-to-bash). Windows users can learn about the Power Shell with _Programming Historian_'s [Introduction to the Windows Command Line with PowerShell](/en/lessons/intro-to-powershell).
 
-You will need any version of Python 3, though 3.5+ is recommended. You will also need the Python package installer, pip.
-A tutorial is available at Programming Historian's [Installing Python Modules with pip](https://programminghistorian.org/en/lessons/installing-python-modules-pip).
+You can use any version of Python 3, though 3.5+ is recommended. You will also need the Python package installer, pip. A tutorial is available at _Programming Historian_'s [Installing Python Modules with pip](/en/lessons/installing-python-modules-pip).
 
 <div class="alert alert-warning">
-Please be aware that as of the date of publication (19 December 2022), the steps for installing PyQt5 in this tutorial will not work with the M-series chipsets used in some newer Mac computers, due to compatibility issues with the required PyQt5 package. Workarounds do exist, but at the moment seem to be system-dependent. If the reader successfully installs PyQt5, the source code in the tutorial will work on M-series chipsets. See the Additional Resources section for more information on troubleshooting installation problems. 
+Please be aware that as of the date of publication (January 2023), the steps for installing PyQt5 in this tutorial will not work with the M-series chipsets used in some newer Mac computers, due to compatibility issues with the required PyQt5 package. Workarounds do exist, but at the moment seem to be system-dependent. If the reader successfully installs PyQt5, the source code in the tutorial will work on M-series chipsets. See the Additional Resources section for more information on troubleshooting installation problems. 
```

Comment:
Hello @DerDoktorFaust and @yann-ryan.

Are both of these warning boxes required? Or would one (combined) suffice?

Reply:
The reason I made two warning boxes is that there are two separate issues (i.e. one about PyQt5 and one about building distributable applications). But, both issues relate to Apple M-series chipsets. 

Would it be better like this:

----start warning box----
For readers with Apple M-Series chipsets:

- PyQt5 warning
- Distributable application warning
----end warning box----

Reply:
I've made this change, seems good to me!